extensions/transform/org.eclipse.smarthome.transform.exec/README.md

```diff
@@ -6,14 +6,45 @@ The external program must either be in the executable search path of the server
 
 ## Example
 
-With the command line 
+**.items**
 
+This will replace the visible label in the UI with the transformation you apply with the command <YourCommand>.
+```java
+String yourItem "Some info  [EXEC(/absolute/path/to/your/TransformProgram %s):]"
 ```
+  
+**.thing**
```

Comment:
While the items- and rules-examples absolutely make sense from my point of view, this one unfortunately does not - the exec binding is an openHAB binding and does not exist here in Eclipse SmartHome.

Reply:
I assume this has been fixed already.

Reply:
No - see https://github.com/eclipse/smarthome/pull/4896#discussion_r161493128